### PR TITLE
11790_home_page 3

### PR DESCRIPTION
**Pull request by tuananhp-1844**
https://edu-redmine.sun-asterisk.vn/issues/11790

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -39,4 +39,9 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function getFullNameAttribute()
+    {
+       return $this->last_name. ' '. $this->first_name;
```
Comment (framgiaciv3) [inline app/Models/User.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Models/User.php</br>**Line**: 45</br>Line indented incorrectly; expected at least 8 spaces, found 7

File: app/Models/Question.php [hunk 2]
```diff
@@ -6,5 +6,33 @@
 
 class Question extends Model
 {
-    //
+    protected $fillable = [
+        'title', 'content', 'user_id', 'category_id', 'status', 'is_poll'
```
Comment (framgiaciv3) [inline app/Models/Question.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ValueNoNewline</br>**Severity**: error</br>**File**: app/Models/Question.php</br>**Line**: 10</br>Each value in a multi-line array must be on a new line

Comment (framgiaciv3) [inline app/Models/Question.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ValueNoNewline</br>**Severity**: error</br>**File**: app/Models/Question.php</br>**Line**: 10</br>Each value in a multi-line array must be on a new line

Comment (framgiaciv3) [inline app/Models/Question.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ValueNoNewline</br>**Severity**: error</br>**File**: app/Models/Question.php</br>**Line**: 10</br>Each value in a multi-line array must be on a new line

Comment (framgiaciv3) [inline app/Models/Question.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ValueNoNewline</br>**Severity**: error</br>**File**: app/Models/Question.php</br>**Line**: 10</br>Each value in a multi-line array must be on a new line

Comment (framgiaciv3) [inline app/Models/Question.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/Question.php</br>**Line**: 10</br>Comma required after last value in array declaration

File: app/Models/User.php [hunk 3]
```diff
@@ -39,4 +39,9 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function getFullNameAttribute()
+    {
+       return $this->last_name.' '.$this->first_name;
```
Comment (framgiaciv3) [inline app/Models/User.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Models/User.php</br>**Line**: 45</br>Line indented incorrectly; expected at least 8 spaces, found 7

**Comment by tuananhp-1844** [comment]
ready

File: resources/views/questions/create.blade.php [hunk 4]
```diff
@@ -0,0 +1,114 @@
+@extends('layouts.app')
+@section('breadcrumbs')
+<div class="breadcrumbs">
+    <section class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <h1>Ask Question</h1>
```
Comment (BunnyPi04) [inline resources/views/questions/create.blade.php]: i18n

**Comment by tuananhp-1844** [comment]
ready

**Comment by tuananhp-1844** [comment]
ready

File: app/Models/User.php [hunk 5]
```diff
@@ -39,4 +39,9 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function getFullNameAttribute()
+    {
+        return $this->last_name.' '.$this->first_name;
```
Comment (thucpv-1748) [inline app/Models/User.php]: convention nhé bạn trẻ

File: resources/views/home.blade.php [hunk 6]
```diff
@@ -1,22 +1,191 @@
...
+                    <p>{{ __('Duis dapibus aliquam mi, eget euismod sem scelerisque ut. Vivamus at elit quis urna adipiscing iaculis. Curabitur vitae velit in neque dictum blandit. Proin in iaculis neque.') }}</p>
+                    <div class="clearfix"></div>
+                    <a class="color button dark_button medium" href="#">{{ __('About Us') }}</a>
+                    <a class="color button dark_button medium" href="#">{{ __('Join Now') }}</a>
                 </div>
+                <div class="col-md-9">
+                    <form class="form-style form-style-2" method="GET" action="{{ route('questions.create') }}">
+                        <p>
+                            <textarea rows="4" id="question_title" readonly
+                                onfocus="if(this.value=={{ __('Ask any question and you be sure find your answer ?') }})this.value='';"
```
Comment (thucpv-1748) [inline resources/views/home.blade.php]: chị Hằng bảo chỗ này bi convention

File: resources/views/home.blade.php [hunk 7]
```diff
@@ -1,22 +1,191 @@
...
+                            <span class="question-answered"><i class="icon-ok"></i>{{ __('in progress') }}</span>
+                            @else
+                            <span class="question-answered question-answered-done"><i class="icon-ok"></i>{{ __('solved') }}</span>
+                            @endif
+                            <span class="question-favorite"><i class="icon-thumbs-up"></i>{{ $question->votes->count() }}</span>
+                        </div>
+                        <span class="question-category"><a href="#"><i class="icon-folder-close"></i>{{ $question->category->name }}</a></span>
+                        <span class="question-date"><i class="icon-time"></i>{{ $question->created_at->diffForHumans() }}</span>
+                        <span class="question-comment"><a href="#"><i class="icon-comment"></i>{{ $question->answers->count() }} {{ __('Answer') }}</a></span>
+                        <span class="question-view"><i class="icon-user"></i>{{$question->view}} {{ __('Views') }}</span>
```
Comment (thucpv-1748) [inline resources/views/home.blade.php]: chị Hằng bảo chỗ này bi convention

File: resources/views/home.blade.php [hunk 8]
```diff
@@ -1,22 +1,191 @@
...
+            <div class="clearfix"></div>
+        </div>
     </div>
-</div>
+
+    <div class="widget widget_stats">
+        <h3 class="widget_title">{{ __('Stats') }}</h3>
+        <div class="ul_list ul_list-icon-ok">
+            <ul>
+                <li><i class="icon-question-sign"></i>{{ __('Questions') }} ( <span>20</span> )</li>
```
Comment (thucpv-1748) [inline resources/views/home.blade.php]: hard code ah

File: resources/views/home.blade.php [hunk 9]
```diff
@@ -1,22 +1,191 @@
...
+            </ul>
+        </div>
+    </div>
+
+    <div class="widget widget_highest_points">
+        <h3 class="widget_title">{{ __('Highest points') }}</h3>
+        <ul>
+            <li>
+                <div class="author-img">
+                    <a href="#"><img width="60" height="60" src="../../ask-me/images/demo/admin.jpg" alt=""></a>
```
Comment (thucpv-1748) [inline resources/views/home.blade.php]: config base url

File: app/Models/User.php (hunk 5 above)
Comment (BunnyPi04) [inline app/Models/User.php]: convention

File: resources/views/home.blade.php (hunk 7 above)
Comment (BunnyPi04) [inline resources/views/home.blade.php]: convention
fix ko hết à?

**Comment by tuananhp-1844** [comment]
ready

File: resources/views/paginations/paginate.blade.php [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+    @endif
+    @for ($i = 1; $i <= $paginator->lastPage(); $i++)
+        @if ($paginator->currentPage() == $i)
+            <span class="current">{{ $i }}</span>
+        @else
+            <a href="{{ $paginator->url($i) }}">{{ $i }}</a>
+        @endif
+    @endfor
+    @if ($paginator->currentPage() !== $paginator->lastPage())
+        <a href="{{ $paginator->url($paginator->currentPage()+1) }}" class="next-button"><i class="icon-angle-right"></i></a>
```
Comment (BunnyPi04) [inline resources/views/paginations/paginate.blade.php]: convention

File: resources/views/questions/show.blade.php [hunk 11]
```diff
@@ -0,0 +1,313 @@
...
+			<div class="question-type-main"><i class="icon-question-sign"></i>{{ __('Question') }}</div>
+		@endif
+        <div class="question-inner">
+            <div class="clearfix"></div>
+            <div class="question-desc">
+                {{ $question->content }}
+            </div>
+            <div class="question-details">
+                @if ($question->is_resolve == 0)
+					<span class="question-answered"><i class="icon-ok"></i>{{ __('in progress') }}</span>
```
Comment (BunnyPi04) [inline resources/views/questions/show.blade.php]: 1 indent = 4 spaces

**Comment by tuananhp-1844** [comment]
ready

**Pull request merged**